### ISSUE TITLE
RendererException: Image format 'ABGR8' is unsupported

### ISSUE THREAD

**Issue by stephengold** [jMonkeyEngine/jmonkeyengine#1272]
As I mentioned at PR #1256, `HelloJME3` with `setGraphicsDebug(true)` crashes quickly on my Linux:

```
Jan 26, 2020 11:58:09 PM com.jme3.app.LegacyApplication handleError
SEVERE: Uncaught exception thrown in Thread[jME3 Main,5,main]
com.jme3.renderer.RendererException: Image format 'ABGR8' is unsupported by the video hardware.
	at com.jme3.renderer.opengl.TextureUtil.getImageFormatWithError(TextureUtil.java:102)
	at com.jme3.renderer.opengl.TextureUtil.uploadTexture(TextureUtil.java:252)
	at com.jme3.renderer.opengl.GLRenderer.updateTexImageData(GLRenderer.java:2498)
	at com.jme3.renderer.opengl.GLRenderer.setTexture(GLRenderer.java:2536)
	at com.jme3.material.Material.updateShaderMaterialParameters(Material.java:845)
	at com.jme3.material.Material.render(Material.java:1020)
	at com.jme3.renderer.RenderManager.renderGeometry(RenderManager.java:614)
	at com.jme3.renderer.queue.RenderQueue.renderGeometryList(RenderQueue.java:266)
	at com.jme3.renderer.queue.RenderQueue.renderQueue(RenderQueue.java:302)
	at com.jme3.renderer.RenderManager.renderViewPortQueues(RenderManager.java:905)
	at com.jme3.renderer.RenderManager.flushQueue(RenderManager.java:779)
	at com.jme3.renderer.RenderManager.renderViewPort(RenderManager.java:1108)
	at com.jme3.renderer.RenderManager.render(RenderManager.java:1166)
	at com.jme3.app.SimpleApplication.update(SimpleApplication.java:272)
	at com.jme3.system.lwjgl.LwjglAbstractDisplay.runLoop(LwjglAbstractDisplay.java:151)
	at com.jme3.system.lwjgl.LwjglDisplay.runLoop(LwjglDisplay.java:197)
	at com.jme3.system.lwjgl.LwjglAbstractDisplay.run(LwjglAbstractDisplay.java:232)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by stephengold** [comment]
And here's the output of `glxinfo` on that same Linux system:
```
Jan 26, 2020 11:58:09 PM com.jme3.app.LegacyApplication handleError
SEVERE: Uncaught exception thrown in Thread[jME3 Main,5,main]
com.jme3.renderer.RendererException: Image format 'ABGR8' is unsupported by the video hardware.
	at com.jme3.renderer.opengl.TextureUtil.getImageFormatWithError(TextureUtil.java:102)
	at com.jme3.renderer.opengl.TextureUtil.uploadTexture(TextureUtil.java:252)
	at com.jme3.renderer.opengl.GLRenderer.updateTexImageData(GLRenderer.java:2498)
	at com.jme3.renderer.opengl.GLRenderer.setTexture(GLRenderer.java:2536)
	at com.jme3.material.Material.updateShaderMaterialParameters(Material.java:845)
	at com.jme3.material.Material.render(Material.java:1020)
	at com.jme3.renderer.RenderManager.renderGeometry(RenderManager.java:614)
	at com.jme3.renderer.queue.RenderQueue.renderGeometryList(RenderQueue.java:266)
	at com.jme3.renderer.queue.RenderQueue.renderQueue(RenderQueue.java:302)
	at com.jme3.renderer.RenderManager.renderViewPortQueues(RenderManager.java:905)
	at com.jme3.renderer.RenderManager.flushQueue(RenderManager.java:779)
	at com.jme3.renderer.RenderManager.renderViewPort(RenderManager.java:1108)
	at com.jme3.renderer.RenderManager.render(RenderManager.java:1166)
	at com.jme3.app.SimpleApplication.update(SimpleApplication.java:272)
	at com.jme3.system.lwjgl.LwjglAbstractDisplay.runLoop(LwjglAbstractDisplay.java:151)
	at com.jme3.system.lwjgl.LwjglDisplay.runLoop(LwjglDisplay.java:197)
	at com.jme3.system.lwjgl.LwjglAbstractDisplay.run(LwjglAbstractDisplay.java:232)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by stephengold** [comment]
Fixed in v3.3 branch at 4c2c3fa3.
